__mocks__/sketch/sample_output.artboard.json

```diff
@@ -1087,6 +1087,5 @@
     "Helvetica-Bold",
     "FiraSans-Regular",
     "FiraSans-Book"
-  ],
-  "sketchVersion": "55"
+  ]
```

Comment:
The sketch version is no longer in the output. You can omit this in Figma since everyone always works against latest version of Figma, but for Sketch we still need this since Sketch updates sometimes contain breaking API updates to tokens.

Please include this again

Reply:
It was included again, appareantly I didn't push the updated sample_output